package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "virtuoso-design-system",
-  "version": "1.11.2",
+  "version": "1.12.0",
```

Comment:
it updates automatically as well. but idk how it determines which part (minor/patch) should be bumped. @jrschumacher help?

Reply:
ah okay, i'll just undo my manual version bump + changelog and let CI handle it automatically

Reply:
@kuguarpwnz @suchak1 we use auto to manage the version. So if you need a minor bump we will apply the `minor` label on the PR. Then the build script will update the changelog and bump.